docs/UserGuide.md

```diff
@@ -161,20 +181,14 @@ Add an ingredient to the Inventory.
     ![addCommand3.png](images/ingredientCommands/addCommand3.png)
 
 **Additional Information:**<br />
+- Refer to the [Command Flags](#42-command-flags) table for information on each `field flag`.
+    - Valid `field flags` are: `-n`, `-q`, `-d`, `-e`, `-t`
 - Any expired ingredients will be automatically tagged as `expired`.
 - Any expiring (within 7 days from current date) will be automatically tagged as `expiring`.
-- For Quantity, units of measurement are not necessary, but the following are accepted:
-    - Acceptable ingredient units are:
-        1. grams: `g`
-        2. litres: `l`
-    - Acceptable prefixes for units are:
-        1. milli- : `m` (i.e. `ml` for millilitres)
-        2. kilo- : `k` (i.e. `kg` for kilograms)
-    - All units will be converted to grams or litres, to 3 decimal places.
 - Please ensure that the units used for quantity are consistent across the Inventory and the RecipeBook if you wish to
-  use the [Cook Recipe](#438-cook-recipe) functionality.
+  use the [Cook Recipe](#458-cook-recipe) functionality.
 
-### 4.2.2 Delete Ingredients
+### 4.4.2 Delete Ingredients
 
 Delete ingredient(s) from the Inventory.
```

Comment:
I feel like the example should stick with one index like `delete ingredient 1` since that is the most common usage.
Then you can say Multidelete is also supported eg. `delete ingredient 1 2 3`